source/settings/enable-dev-tools.txt

```diff
@@ -0,0 +1,106 @@
+.. _compass-enable-dev-tools:
+
+======================
+Toggle Chrome DevTools
+======================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can toggle the ``enableDevTools`` option to control user access to Chrome 
+DevTools on |compass|. 
+
+About This Task
+---------------
+
+By default, :guilabel:`Enable DevTools` is enabled.
+
+If you set :guilabel:`Set Read-Only Mode`, :guilabel:`Enable MongoDB Shell`, 
```

Comment:
**Suggestion**

Slight tweak to make this more readable with a bulleted list:

Compass automatically disables the DevTools setting if you  enable any of the following: 

- Read-Only Mode
- The Compass MongoDB Shell
- maxTimeMS
- Protect Connection String Secrets

Reply:
Oh great suggestion!! I was having trouble parsing it myself but didn't think of that 😅